### PR TITLE
Added filtering to device table

### PR DESCRIPTION
**Pull request by TomasHodor**
Signed-off-by: thodor <thodor@frinx.io>

### CODE REVIEW

File: src/components/uniconfig/deviceTable/List.js [hunk 1]
```diff
@@ -390,10 +412,10 @@ class List extends Component {
                             <thead>
                                 <tr>
                                     <th>Select</th>
-                                    <th>Node ID</th>
-                                    <th>IP address</th>
-                                    <th>Status</th>
-                                    <th>OS/Version</th>
+                                    <th className="tableHeader" onClick={(e) => this.sort(e,0)}>Node ID</th>
```
Comment (ChmaraX) [inline src/components/uniconfig/deviceTable/List.js]: Why passing "e" in the function?

File: src/components/uniconfig/deviceTable/List.js [hunk 2]
```diff
@@ -363,6 +365,26 @@ class List extends Component {
...
+        this.state.keywords === ""
+            ? this.setState({
+                data: dataset,
+                sort: !sort
+            })
+            : this.setState({
+                table: dataset,
+                sort: !sort
+            })
+    }
```
Comment (ChmaraX) [inline src/components/uniconfig/deviceTable/List.js]: this.setState({
            [this.state.keywords === "" ? "data" : "table"]: dataset,
            sort: !sort
        })

File: src/components/uniconfig/deviceTable/List.js (hunk 1 above)
Comment (TomasHodor) [reply src/components/uniconfig/deviceTable/List.js]: Done

File: src/components/uniconfig/deviceTable/List.js (hunk 2 above)
Comment (TomasHodor) [reply src/components/uniconfig/deviceTable/List.js]: Done

**Pull request merged**